components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -4296,6 +4296,36 @@ def searchProcessing():
             search.close()
         return rv
 
+    def getThumbnailSet(self, image_ids, max_size=64):
```

Comment:
default is 64 https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroPy/src/omero/gateway/__init__.py#L7781